### DISCUSSION TITLE
I'm confused how to activate this

### DISCUSSION THREAD

**Discussion by cousc0us** [Hubro/clipface#36] [Q&A] [answered]
hi, im new to github so i dont rrly know stuff ;-;. I downloaded the file but I don't know what to do, possible to help me?

**Comment by Hubro** [comment, marked as answer]
The simplest way to deploy Clipface is using Docker.

If you've never heard of Docker before, they have a good [Get Started](https://docs.docker.com/get-started/) guide here. After that, read the [Usage with Docker](https://github.com/Hubro/clipface#usage-with-docker) section of the Clipface README file.

If you have any issues deploying Clipface after that, let me know here.

**Reply by cousc0us** [reply]
Thanks, I launched it. But how do I insert my clips? I can't find an insert button ;-; (srry im new)

**Reply by Hubro** [reply]
You don't insert clips via the web interface, you just place clips into a directory that Clipface watches. If you check the simple Docker example on the front page:

```sh
docker run -d \
  --name clipface \
  -v /host/path/to/clips:/clips \
  -p 80:80 \
  tomsan/clipface:latest
```

The `-v /host/path/to/clips:/clips` argument maps a directory from the host computer into the Docker container. Clipface inside the Docker container looks for clips inside the `/clips` directory. If you have your clips in the folder `/home/herothing/Video/Clips` on the host machine, you can map that folder into the Docker container using this argument: `-v /home/herothing/Video/Clips:/clips`. That will make Clipface display those clips.

**Reply by cousc0us** [reply]
I don't really understand, what if I put my videos in a folder inside my downloads folder? and how can i find the /host/path/to/clip folder?

**Reply by Hubro** [reply]
I'm struggling to understand what you are asking.

If you have a folder on your server or computer called (for example) `/home/herothing428/Downloads/Clips`, you can launch Clipface like this:

```
docker run -d \
  --name clipface \
  -v /home/herothing428/Downloads/Clips:/clips \
  -p 80:80 \
  tomsan/clipface:latest
```

Now you can place clips inside the `/home/herothing428/Downloads/Clips` folder and they will be displayed in Clipface.

**Reply by therealgrimshader** [reply]
I think they mean can they map the clips folder to their downloads folder.
